pyproject.toml

```diff
@@ -24,6 +24,8 @@ before-build = [
     "{project}/Finch-mlir/scripts/docker_prepare_ccache.sh"
 ]
 environment-pass = ["HOST_CCACHE_DIR", "SPARSE_BACKEND"]
+manylinux-x86_64-image = "manylinux_2_28"
+manylinux-pypy_x86_64-image = "manylinux_2_28"
```

Comment:
Why do we need this change?

Reply:
I'm trying to get pixi to work -- no idea if it will though. 😢

Reply:
I think I prefer to leave it open for now. We have working wheels now I would avoid interfering with build process to not to spend more time on this.

Once we have MVP of Finch mlir dialect and we need to iterate on the build process once again, we can include it.